src/sentry/static/sentry/app/views/settings/organizationMembers/inviteMember/teamSelect.jsx

```diff
@@ -3,31 +3,40 @@ import React from 'react';
 import styled from 'react-emotion';
 
 import {t} from 'app/locale';
+import Button from 'app/components/button';
 import Checkbox from 'app/components/checkbox';
 import IdBadge from 'app/components/idBadge';
 import {Panel, PanelBody, PanelHeader, PanelItem} from 'app/components/panels';
 
-const TeamItem = styled.div`
-  width: 25%;
-  padding: 6px;
-`;
-
 class TeamSelect extends React.Component {
   static propTypes = {
     disabled: PropTypes.bool,
     selectedTeams: PropTypes.instanceOf(Set),
     teams: PropTypes.array,
     toggleTeam: PropTypes.func,
+    onSelectAll: PropTypes.func,
   };
 
   render() {
-    let {disabled, teams, selectedTeams, toggleTeam} = this.props;
+    let {disabled, teams, selectedTeams, toggleTeam, onSelectAll} = this.props;
     //no need to select a team when there's only one option
     if (teams.length < 2) return null;
 
     return (
       <Panel className="new-invite-team">
-        <PanelHeader>{t('Team')}</PanelHeader>
+        <PanelHeader hasButtons>
+          {t('Team')}
+          {onSelectAll && (
+            <Button
+              className="select-all"
+              size="small"
+              disabled={disabled}
+              onClick={() => onSelectAll()}
```

Comment:
I don't think the closure here is necessary

```suggestion
              onClick={onSelectAll}
```